PRIVACY.md

```diff
@@ -0,0 +1,21 @@
+# Semgrep Privacy Policy
+
+Semgrep collects usage data to help us improve the product. We send two types of data to r2c servers: scan data and findings data.
```

Comment:
Semgrep collects usage data to provide useful results and to help us improve the product. We send two types of data to r2c servers: scan data and findings data.